### PR TITLE
TST: added test for to_json when called on numbers exceeding the int64 limit

### LINKED ISSUE #21073: pandas dataframe to_json(orient='split') change uint64 to negative int
#### Code Sample

```python
import pandas
import json
d = {'col1': [13342205958987758245, 12388075603347835679], 'col2': [13342204859476130034, 12388075603347835679]}
df = pandas.DataFrame(data=d)

print (df)
print (df.dtypes)
print ("--- to json output")

print (df.head(5).fillna('NULL').to_json(orient='split'))

print ("--- to dict output")
print (json.dumps(df.head(5).fillna('NULL').to_dict(orient='split')))

```

#### Problem description
the output of the above code is : 
```
                   Col1                  Col2
0  13342205958987758245  12388075603347835679
1  13342204859476130034  12388075603347835679

--- dtypes
Col1    uint64
Col2    uint64
dtype: object
--- to json output
{"columns":["Col1","Col2"],"index":[0,1,2,3,4],"data":[[-5104538114721793371,-6058668470361715937],[-5104539214233421582,-6058668470361715937],[-5104583194698550022,-6058668470361715937],[-3637227675748224855,-6058668470361715937],[-6228767139342072872,-6058667370850087726]]}
--- to dict output
{"index": [0, 1, 2, 3, 4], "data": [[13342205958987758245, 12388075603347835679], [13342204859476130034, 12388075603347835679], [13342160879011001594, 12388075603347835679], [14809516397961326761, 12388075603347835679], [12217976934367478744, 12388076702859463890]], "columns": ["Col1", "Col2"]}
```

you can see the data are different for these two similar functions: `to_json` and `to_dict`.
I believe that the output of `to_dict`is correct but `to_json` has change my original digits.

#### Expected Output
The expected output is what we see in the `to_dict` output.

#### Output of ``pd.show_versions()``

<details>

[paste the output of ``pd.show_versions()`` here below this line]
>>> pd.show_versions()

INSTALLED VERSIONS
------------------
commit: None
python: 2.7.14.final.0
python-bits: 64
OS: Darwin
OS-release: 16.7.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: None
LANG: None
LOCALE: None.None

pandas: 0.22.0
pytest: None
pip: 9.0.1
setuptools: 38.5.0
Cython: None
numpy: 1.14.0
scipy: None
pyarrow: None
xarray: None
IPython: None
sphinx: None
patsy: None
dateutil: 2.6.1
pytz: 2017.3
blosc: None
bottleneck: None
tables: None
numexpr: None
feather: None
matplotlib: None
openpyxl: None
xlrd: 1.0.0
xlwt: 1.3.0
xlsxwriter: 1.0.2
lxml: None
bs4: None
html5lib: None
sqlalchemy: 0.8.0
pymysql: None
psycopg2: None
jinja2: 2.7.1
s3fs: None
fastparquet: None
pandas_gbq: None
pandas_datareader: None
</details>

### PR DESCRIPTION
**Pull request by johnmantios**
- [x] closes #21073
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) 
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Added test for to_json() when called on numbers exceeding the int64 limit.
Tests and linter pass locally.

### CODE REVIEW

File: pandas/tests/io/json/test_pandas.py [hunk 1]
```diff
@@ -1908,3 +1908,20 @@ def test_complex_data_tojson(self, data, expected):
...
+    )
+    @pytest.mark.parametrize(
+        "expected",
+        [
+            '{"columns":["col1"],"index":[0,1],'
+            '"data":[[13342205958987758245],[12388075603347835679]]}'
+        ],
+    )
+    @pytest.mark.parametrize("orient", ["split"])
+    def test_json_uint64(self, data, expected, orient):
```
Comment (phofl) [inline pandas/tests/io/json/test_pandas.py:1923]: This is only one test right? You don’t have to paeametrize over arguments that take only one value

Comment (johnmantios) [reply pandas/tests/io/json/test_pandas.py:1923]: oh ok. Should I refactor and resubmit?

Comment (phofl) [reply pandas/tests/io/json/test_pandas.py:1923]: You can just update this brancj

File: pandas/tests/io/json/test_pandas.py [hunk 2]
```diff
@@ -1908,3 +1908,15 @@ def test_complex_data_tojson(self, data, expected):
...
         assert result == expected
+
+    def test_json_uint64(self):
+        # GH21073
+        data = {"col1": [13342205958987758245, 12388075603347835679]}
+        expected = (
+            '{"columns":["col1"],"index":[0,1],'
+            '"data":[[13342205958987758245],[12388075603347835679]]}'
+        )
+        orient = "split"
```
Comment (phofl) [inline pandas/tests/io/json/test_pandas.py:1919]: You don't have to define orient or data as variables, just put them into the functions directly

Comment (johnmantios) [reply pandas/tests/io/json/test_pandas.py]: Thanks for the feedback! This is my first contribution to open source, so bear with me :)

**Review by phofl — APPROVED** [review]

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @johnmantios

**Pull request merged**